### ISSUE TITLE
Special input mask to go through all files still to be described at once / notification icon

### ISSUE THREAD

**Issue by svewap** [marketing-factory/ai-filemetadata#10]
The results of the automatic description should be approved by the editors. A special input screen that only shows the image and a few input fields would be helpful. There is also a huge pile of to-dos at the beginning. Editors could be informed that descriptions are still open by means of a notification icon in the top bar.

**Comment by vitamys** [comment]
we are interested in this. would you accept a PR for a BE Module?

**Comment by ischmittis** [comment]
@vitamys yes. But it's not only a BE module. We need to change toe complete logic, so that the AI generated Text is written into a separated field, so we can show it and approve it. If you have any ideas on how to implement - we are happy for your PR.

**Comment by vitamys** [comment]
I was thinking of a view to show the editor what has been done - so they can check and in case they dont agree, they can change the text. so generated text goes into the alt field, and is shown to editor in the module, where they can edit it if needed. maybe add a field 'reviewed by editor' .
Also, that they can generate AI texts from this module, especially when the ext is introduced.

**Comment by ischmittis** [comment]
That is partly already possible by TYPO3 Core in file module. "Select colums to view in filelist." But this is only basic. A specific module would be perfect!
